mathics/builtin/graphics.py

```diff
@@ -694,11 +777,45 @@ def apply(self, c1, c2, evaluation, options):
             if not compute:
                 evaluation.message('ColorDistance', 'invdist', distance_function)
                 return
+        elif distance_function.has_form('List', 2):
+            if (isinstance(distance_function.leaves[0], String)
+            and distance_function.leaves[0].get_string_value() == 'CMC'):
+                if isinstance(distance_function.leaves[1], String):
+                    if distance_function.leaves[1].get_string_value() == 'Acceptability':
+                        compute = lambda c1, c2: _CMC_distance(100*c1.to_color_space('LAB')[:3],
+                                                              100*c2.to_color_space('LAB')[:3], 2, 1)/100
+                    elif distance_function.leaves[1].get_string_value() == 'Perceptibility':
+                        compute = ColorDistance._distances.get("CMC")
+                    else:
+                        evaluation.message('ColorDistance', 'invdist', distance_function)
+                        return
+                    
+                if (distance_function.leaves[1].has_form('List', 2)
+                and distance_function.leaves[1].leaves[0].get_int_value() > 0
+                and distance_function.leaves[1].leaves[1].get_int_value() > 0):
```

Comment:
calling `get_int_value` here is potentially unsafe because we don't know the leaves are `Integer`s.